spec/requests/user_creates_review_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'spec_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.